src/mpfao/tdympfao.c

```diff
@@ -440,6 +467,7 @@ PetscErrorCode TDyMPFAOInitialize(TDy tdy) {
   // why must these be placed after SetPermeabilityFromFunction()?
   ierr = ComputeGMatrix(tdy); CHKERRQ(ierr);
```

Comment:
At some point, it might make sense for us to break state data out of the model itself. It's very strange to just call a bunch of methods on an object and hope that it's in the right state before/afterward.